### PR TITLE
Moved more traits & activities to Mods.Common

### PR DESCRIPTION
**Pull request by reaperrr**
Mostly infantry and building traits.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Guardable.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2014 The OpenRA Developers (see AUTHORS)
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Guardable.cs]: Make this 2015 while you are at it.

Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/Guardable.cs]: Ah yes, good point.

**Comment by Mailaender** [comment]
:+1: on the movings.

File: OpenRA.Mods.Common/Traits/Guard.cs [hunk 2]
```diff
@@ -14,15 +14,14 @@
...
 
-namespace OpenRA.Mods.RA.Traits
+namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("The player can give this unit the order to follow and protect friendly units with the Guardable trait.")]
-	class GuardInfo : TraitInfo<Guard> { }
+	public class GuardInfo : TraitInfo<Guard> { }
 
-	class Guard : IResolveOrder, IOrderVoice
+	public class Guard : IResolveOrder, IOrderVoice
```
Comment (penev92) [inline OpenRA.Mods.Common/Traits/Guard.cs]: Can we rename this to `Guards` ?
`Guard` sounds more like like an activity than a trait.

**Comment by penev92** [comment]
:+1:

**Pull request merged**